### PR TITLE
API: update user post

### LINKED ISSUE #21: API: Update user post
- User story #8

### PR DESCRIPTION
**Pull request by benedekhalaj**
- Closes #21

### CODE REVIEW

File: springboot/src/test/java/com/codecool/terraformingmarsforum/controller/UserPostControllerTest.java [hunk 1]
```diff
@@ -66,4 +78,37 @@ public void convertToUserPost_ConvertingCreateUserPostRequestToUserPost_CreatesU
...
     }
 
-}
\ No newline at end of file
+    @Test
+    public void createUserPost_CreatingUserPost_HasStatus201() {
+        UserPostController.CreateUserPostRequest createUserPostRequest = getCreateUserPostRequest();
+        UserPost userPost = createUserPostRequest.convertToUserPost();
+
+        when(userPostService.createUserPost(any(UserPost.class))).thenReturn(userPost);
```
Comment (dudaskobendeguz) [inline springboot/src/test/java/com/codecool/terraformingmarsforum/controller/UserPostControllerTest.java:86]: I like this solution to mock Entity class with null id

**Review by dudaskobendeguz — APPROVED** [review]
I love to see these tests(@Zsezsu has more, but it's okay :+1: )

**Comment by dudaskobendeguz** [comment]
I love each and every of your tests :lab_coat: :baby_bottle: :cactus: :face_with_spiral_eyes: :t-rex:

**Comment by benedekhalaj** [comment]
> I love to see these tests(@Zsezsu has more, but it's okay +1 )

I am about to find out... haha

**Pull request merged**